extensions/amp-sidebar/1.0/amp-sidebar-1.0.md

```diff
@@ -58,103 +62,240 @@ limitations under the License.
 - Touch zoom is disabled on the `amp-sidebar` and it's mask when the sidebar is open.
 
 Example:
+
 ```html
-<amp-sidebar id='sidebar1' layout='nodisplay'>
+<amp-sidebar id="sidebar1" layout="nodisplay">
   <ul>
     <li> Nav item 1</li>
     <li> Nav item 2</li>
     <li> Nav item 3</li>
     <li> Nav item 4</li>
     <li> Nav item 5</li>
     <li> Nav item 6</li>
-    <li> Nav item 7</li>
-    <li> Nav item 8</li>
-    <li> Nav item 9</li>
-    <li on="tap:sidebar1.close"> Close</li>
   </ul>
 </amp-sidebar>
 ```
 
-### Opening and Closing the Sidebar
-Setting the `on` attribute on one or more elements within the page and setting it's method to `toggle` will toggle the sidebar when the element is tapped or clicked. Setting the element's method to `open` or `close` will open or close the sidebar.Tapping back on the partially-visible main content area closes the sidebar.
+### Opening and closing the sidebar
 
-Alternatively pressing the escape key on the keyboard will also close the sidebar.
+To toggle, open, or close the sidebar when an element is tapped or clicked, set the [`on`](../../../spec/amp-actions-and-events.md) action attribute on the element, and specify one of the following action methods:
+
+<table>
+  <tr>
+    <th>Action</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>open (default)</td>
+    <td>Opens the sidebar</td>
+  </tr>
+  <tr>
+    <td>close</td>
+    <td>Closes the sidebar</td>
+  </tr>
+  <tr>
+    <td>toggle</td>
+    <td>Toggles the sidebar state</td>
+  </tr>
+</table>
+
+If the user taps back on the partially-visible main content area, this closes the sidebar.
+
+Alternatively, pressing the escape key on the keyboard will also close the sidebar.
 
 Example:
+
 ```html
 <button class="hamburger" on='tap:sidebar1.toggle'></button>
 <button on='tap:sidebar1'>Open</button>
 <button on='tap:sidebar1.open'>Open</button>
 <button on='tap:sidebar1.close'>x</button>
 ```
 
-## Attributes
+### Toolbar
 
-**side**
+You can create a toolbar element that displays in the `<body>` by specifying the toolbar attribute with a media query on a `<nav>` element that is a child of  `<amp-sidebar>`. The toolbar duplicates the `<nav>` element and its children.
```

Comment:
nit: put `toolbar` in code format (ie., use backticks) - this is for the first one where it says "`toolbar` attribute".